yt/yt/ytlib/query_tracker_client/helpers.cpp

```diff
@@ -51,4 +58,132 @@ bool IsFinishedState(EQueryState state)
 
 ////////////////////////////////////////////////////////////////////////////////
 
+void ToProto(NProto::TQuery* protoQuery, const NApi::TQuery& query)
+{
+    protoQuery->Clear();
+
```

Comment:
Can we do `static_assert(NProto::TQuery::FieldsCount == 42)` and `static_assert::(NApi::TQuery::FieldCount == 654)` so people don't forget to update this part.

Reply:
Removed this part